dap4_test/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ IF(ENABLE_DAP_REMOTE_TESTS)
     # Change name (add '4') to avoid cmake
     # complaint about duplicate targets.
     BUILD_BIN_TEST(findtestserver4)
+    BUILD_BIN_TEST(pingurl4)
```

Comment:
This fails due to the lack of a `pingurl4.c` file in the `dap4_test/` directory. There are workarounds for this.  I'm merging this PR into a branch from the local trunk so that I can add the fix. Thanks @DennisHeimbigner !